catalog/OpenET/OpenET_ENSEMBLE_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet

```diff
@@ -0,0 +1,261 @@
+local id = 'OpenET/ENSEMBLE/CONUS/GRIDMET/MONTHLY/v2_0';
+local subdir = 'OpenET';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
```

Comment:
please add at the top level:
'gee:skip_indexing': true,
'gee:user_uploaded': true,

These are temporary flags the process currently requires.

Reply:
I did this in #305, so it's all set.